doc/install_generic.inc

```diff
@@ -9,6 +9,8 @@ Installation
 Stable Installation
 -------------------
 
+.. comment: install_theano_stable_version
+
 Install the latest stable version of Theano with:
```

Comment:
I think it would be worth putting the installation instructions for the conda packages for theano and libgpuarray here, actually, not only on Windows.